climpred/bias_reduction.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+import xarray as xr
+
+from climpred.metrics import Metric
+
+
+def _mean_bias_reduction_quick(hind, bias, dim):
+    """Quick reduction of mean bias over all initializations.
+
+    Args:
+        hind (xr.object): hindcast.
+        bias (xr.object): bias.
+        dim (str): Time dimension name in bias.
+
+    Returns:
+        xr.object: bias reduced hind
+
+    """
+    bias_reduced_hind = hind - bias.mean(dim)
+    return bias_reduced_hind
+
+
+def _mean_bias_reduction_properly(hind, bias, dim):
+    """Reduce mean bias from all but the given initialization (cross-validation).
+
+    Args:
+        hind (xr.object): hindcast.
+        bias (xr.object): bias.
+        dim (str): Time dimension name in bias.
+
+    Returns:
+        xr.object: bias reduced hind
+
+    Reference:
+    - Jolliffe, Ian T., and David B. Stephenson. Forecast Verification:
+      A Practitioner’s Guide in Atmospheric Science. Chichester, UK: John Wiley & Sons,
+      Ltd, 2011. https://doi.org/10.1002/9781119960003., Chapter: 5.3.1, p.80
+
+    """
+    bias = bias.rename({dim: 'init'})
+    bias_reduced_hind = []
+    logging.info('mean bias reduction:')
+    for init in hind.init.values:
+        hind_drop_init = hind.init.drop_sel(init=init)
+        hind_drop_init_where_bias = hind_drop_init.where(bias.init)
```

Comment:
Is `drop=True` default now? It seems like this is dropping appropraitely. Maybe write `drop=True` to make it more explicit here.

Reply:
no its not